### DISCUSSION TITLE
Is there a tree-table widget?

### DISCUSSION THREAD

**Discussion by AlexKotik** [Kitware/trame#446] [Q&A]
I'd like to show to a user some hierarchical tables with multiple levels of nesting. But it seems that neither Vuetify nor Quasar nor Formkit have a built-in one. Do you know if that kind of widget already exist and integrated in Trame? Or I'm stuck with building it myself? To make it clearer, I'm talking about something like this:
![1_ARieJYsu_7Kgs5sPmRXcqw](https://github.com/Kitware/trame/assets/24991384/e48fc0c2-ccc7-4d43-8e51-b701fd4afb7b)

**Comment by jourdain** [comment]
I'm not aware of any, but any vue.js library out there, if it exists, [can be integrated into Trame fairly easily](https://trame.readthedocs.io/en/latest/tools.widgets.html).

**Comment by jourdain** [comment]
I haven't looked, but maybe https://github.com/Kitware/trame-datagrid does provide something like that?
